### PR TITLE
Fix/new: Fixed add new form in product entry page

### LINKED ISSUE #90: Product entry does not work at all
When Add new Product entry is click, page breaks

Fix the Add new aProduct entry step

When a user clicks on new product entry, it should show the product entry page 

After user enters all the needed information and clicks on Add new

Product should be aded succesfully and shown in the Product List

### PR DESCRIPTION
**Pull request by inuoluwadunsimi**
closes #90 
Screenshot (67)

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -64,4 +64,4 @@ module.exports = {
       },
     },
   },
-}
+};
```
Comment (HealthStack01) [inline .eslintrc.js:67]: Change not required 🧹

File: src/pages/routes.tsx [hunk 2]
```diff
@@ -91,6 +91,7 @@ const AppRoutes = () => {
           />
           <Route path="/app/clients/clients" element={<AppClient />} />
 
+
```
Comment (HealthStack01) [inline src/pages/routes.tsx:94]: Change not required 🧹

File: src/pages/app/pharmacy/ProductEntry/index.tsx [hunk 3]
```diff
@@ -66,6 +66,8 @@ const AppProductEntry = () => {
           selectedData={selectedProductEntry}
         />
       )}
+    
+
```
Comment (HealthStack01) [inline src/pages/app/pharmacy/ProductEntry/index.tsx:70]: 🧹

**Review by ghost — CHANGES_REQUESTED** [review]
Make only required and related changes, Also add description and snapshots 🤞

**Review by ghost — APPROVED** [review]

**Pull request closed without merging**